CHANGELOG.md

```diff
@@ -3,3 +3,23 @@
 ## 1.0.0 (Unreleased)
 
 Initial release.
+
+## 0.4.1
+
+Improved curated dashboard.
+
+## 0.4.0
+
+Allow to authenticate using AWS Secret Manager. More bug fixes.
```

Comment:
nit: would be good to highlight what the bug fixes were so it's easier for people to search in the CHANGELOG (it doesn't matter with this release because no one has a previous version 😂  but for future releases past v1.0.0, it would be good 👍 )

Reply:
I agree, these release were internal, that's why they didn't have so much detail. Thanks!